xline/src/main.rs

```diff
@@ -162,43 +163,49 @@ struct ServerArgs {
     /// Open jaeger online
     #[clap(long)]
     jaeger_online: bool,
-    /// Trace level of jaeger
+    /// Trace level of  [default: info]
     #[clap(long, value_parser = parse_log_level, default_value_t = default_log_level())]
     jaeger_level: LevelConfig,
     /// Log file path
     #[clap(long, default_value = "/var/log/xline")]
     log_file: PathBuf,
-    /// Log rotate strategy, eg: never, hourly, daily
+    /// Log rotate strategy, eg: never, hourly, daily, [default: daily]
     #[clap(long, value_parser = parse_rotation, default_value_t = default_rotation())]
     log_rotate: RotationConfig,
-    /// Log verbosity level, eg: trace, debug, info, warn, error
+    /// Log verbosity level, eg: trace, debug, info, warn, error, [default: info]
     #[clap(long, value_parser = parse_log_level, default_value_t = default_log_level())]
     log_level: LevelConfig,
-    /// Heartbeat interval between curp server nodes
+    /// Heartbeat interval between curp server nodes [default: 300ms]
     #[clap(long, value_parser = parse_duration)]
     heartbeat_interval: Option<Duration>,
-    /// Curp wait sync timeout
+    /// Curp wait sync timeout [default: 5s]
     #[clap(long, value_parser = parse_duration)]
     server_wait_synced_timeout: Option<Duration>,
-    /// Curp propose retry timeout
+    /// Curp propose retry timeout [default: 50ms]
     #[clap(long, value_parser = parse_duration)]
     retry_timeout: Option<Duration>,
-    /// Curp rpc timeout
+    /// Curp rpc timeout [default: 50ms]
     #[clap(long, value_parser = parse_duration)]
     rpc_timeout: Option<Duration>,
+    /// Curp append entries batch timeout [default: 1ms]
+    #[clap(long, value_parser = parse_duration)]
+    batch_timeout: Option<Duration>,
+    /// Curp append entries batch max size [default: 2MB]
+    #[clap(long, value_parser = parse_batch_bytes)]
+    batch_max_size: Option<u64>,
     /// Follower election timeout ticks
```

Comment:
Why is this comment not updated

Reply:
That's because the default value of follower election timeout ticks is set by `default_value_t`, we don't need to describe it in the comment.